dev-lang/go/go-1.5.ebuild

```diff
@@ -0,0 +1,106 @@
+# Copyright 1999-2015 Gentoo Foundation
```

Comment:
Use the name `go-9999.ebuild` since this is live. If you want to snapshot a particular version you can use something like `go-1.5_pre20150515` and set EGIT_COMMIT=<hash> in the ebuild.